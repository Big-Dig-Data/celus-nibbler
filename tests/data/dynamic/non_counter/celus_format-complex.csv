"Item","Metric","Institution","Year of publication","ISBN","eissn","Jan 2019","Feb 2019","Mar 2019","Country"
"I1","M1","O1","2001","111-1-1111-1111-1","1111-1111",1,2,3,"us"
"I1","M2","O1","2001","111-1-1111-1111-1","1111-1111",2,3,4,"us"
"I1","M1","O1","2001","111-1-1111-1111-1","1111-1111",5,6,7,"cs"
"I1","M2","O1","2001","111-1-1111-1111-1","1111-1111",8,9,10,"cs"
"I2","M1","O1","2002","222-2-2222-2222-2","2222-2222",11,12,13,"us"
"I2","M2","O1","2002","222-2-2222-2222-2","2222-2222",12,13,14,"us"
"I2","M1","O1","2002","222-2-2222-2222-2","2222-2222",15,16,17,"cs"
"I2","","O1","2002","222-2-2222-2222-2","2222-2222",18,19,20,"cs"
"I1","M1","O2","2001","111-1-1111-1111-1","1111-1111",9,8,7,"us"
"I1","M2","O2","2001","111-1-1111-1111-1","1111-1111",8,7,6,"us"
"I1","M1","O2","2001","111-1-1111-1111-1","1111-1111",7,6,5,"cs"
"I1","M2","O2","2001","111-1-1111-1111-1","1111-1111",6,5,4,"cs"
"I3","M1","O2","2003","333-3-3333-3333-3","3333-3333",5,4,3,"us"
"I3","M2","O2","2003","333-3-3333-3333-3","3333-3333",4,3,2,"us"
"I3","M1","O2","2003","333-3-3333-3333-3","3333-3333",3,2,1,"cs"
"I3","M2","O2","2003","333-3-3333-3333-3","3333-3333",2,1,0,"cs"
