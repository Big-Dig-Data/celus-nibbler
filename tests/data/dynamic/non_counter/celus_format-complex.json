{
  "kind": "non_counter.celus_format",
  "version": 1,
  "parser_name": "tabular",
  "data_format": {
    "name": "Complex"
  },
  "platforms": [
    "Platform Complex"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": [],
  "metric_aliases": [
      ["MD", "Metric default"]
  ],
  "dimension_aliases": [
      ["YOP", "YearOfPublish"]
  ],
  "areas": [
    {
      "kind": "non_counter.celus_format",
      "title_column_names": ["Item"],
      "metric_column_names": ["Metric"],
      "default_metric": "MD",
      "organization_column_names": ["Institution"],
      "dimension_mapping": {
        "Year of publication": "YOP",
        "Country": "Country"
      },
      "title_ids_mapping": {
        "ISBN": "ISBN",
        "eissn": "Online_ISSN"
      },
      "value_extract_params": {
         "default": 0
      }
    }
  ]
}
