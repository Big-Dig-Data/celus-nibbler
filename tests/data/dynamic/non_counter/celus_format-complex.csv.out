"2019-01-01","2019-01-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M1","1"
"2019-02-01","2019-02-28","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M1","2"
"2019-03-01","2019-03-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M1","3"
"2019-01-01","2019-01-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M2","2"
"2019-02-01","2019-02-28","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M2","3"
"2019-03-01","2019-03-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M2","4"
"2019-01-01","2019-01-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M1","5"
"2019-02-01","2019-02-28","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M1","6"
"2019-03-01","2019-03-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M1","7"
"2019-01-01","2019-01-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M2","8"
"2019-02-01","2019-02-28","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M2","9"
"2019-03-01","2019-03-31","O1","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M2","10"
"2019-01-01","2019-01-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:us|YearOfPublish:2002","M1","11"
"2019-02-01","2019-02-28","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:us|YearOfPublish:2002","M1","12"
"2019-03-01","2019-03-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:us|YearOfPublish:2002","M1","13"
"2019-01-01","2019-01-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:us|YearOfPublish:2002","M2","12"
"2019-02-01","2019-02-28","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:us|YearOfPublish:2002","M2","13"
"2019-03-01","2019-03-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:us|YearOfPublish:2002","M2","14"
"2019-01-01","2019-01-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:cs|YearOfPublish:2002","M1","15"
"2019-02-01","2019-02-28","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:cs|YearOfPublish:2002","M1","16"
"2019-03-01","2019-03-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:cs|YearOfPublish:2002","M1","17"
"2019-01-01","2019-01-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:cs|YearOfPublish:2002","Metric default","18"
"2019-02-01","2019-02-28","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:cs|YearOfPublish:2002","Metric default","19"
"2019-03-01","2019-03-31","O1","I2","ISBN:222-2-2222-2222-2|Online_ISSN:2222-2222","","","Country:cs|YearOfPublish:2002","Metric default","20"
"2019-01-01","2019-01-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M1","9"
"2019-02-01","2019-02-28","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M1","8"
"2019-03-01","2019-03-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M1","7"
"2019-01-01","2019-01-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M2","8"
"2019-02-01","2019-02-28","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M2","7"
"2019-03-01","2019-03-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:us|YearOfPublish:2001","M2","6"
"2019-01-01","2019-01-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M1","7"
"2019-02-01","2019-02-28","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M1","6"
"2019-03-01","2019-03-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M1","5"
"2019-01-01","2019-01-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M2","6"
"2019-02-01","2019-02-28","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M2","5"
"2019-03-01","2019-03-31","O2","I1","ISBN:111-1-1111-1111-1|Online_ISSN:1111-1111","","","Country:cs|YearOfPublish:2001","M2","4"
"2019-01-01","2019-01-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:us|YearOfPublish:2003","M1","5"
"2019-02-01","2019-02-28","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:us|YearOfPublish:2003","M1","4"
"2019-03-01","2019-03-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:us|YearOfPublish:2003","M1","3"
"2019-01-01","2019-01-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:us|YearOfPublish:2003","M2","4"
"2019-02-01","2019-02-28","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:us|YearOfPublish:2003","M2","3"
"2019-03-01","2019-03-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:us|YearOfPublish:2003","M2","2"
"2019-01-01","2019-01-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:cs|YearOfPublish:2003","M1","3"
"2019-02-01","2019-02-28","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:cs|YearOfPublish:2003","M1","2"
"2019-03-01","2019-03-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:cs|YearOfPublish:2003","M1","1"
"2019-01-01","2019-01-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:cs|YearOfPublish:2003","M2","2"
"2019-02-01","2019-02-28","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:cs|YearOfPublish:2003","M2","1"
"2019-03-01","2019-03-31","O2","I3","ISBN:333-3-3333-3333-3|Online_ISSN:3333-3333","","","Country:cs|YearOfPublish:2003","M2","0"
